5/5.6/download_source

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+#
+# Copyright (C) 2017 Google Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+SCRIPT_DIR=$(readlink -f $(dirname "${BASH_SOURCE[0]}"))
+cd "$SCRIPT_DIR"
```

Comment:
Ignore this if we move it to `/tmp` as I suggested.

Could you add a comment what it does? I have needed some time to figure it out.

My suggestion of simplification:

`# Some short description`
`cd "$(dirname "${BASH_SOURCE[0]}")"`